### DISCUSSION TITLE
Networking unreachable for LInux Ubuntu 22.02 guest after installing UTM Version 4.0.8 (67)

### DISCUSSION THREAD

**Discussion by canoe-nc** [utmapp/UTM#4543] [UTM Help (macOS)] [answered]
Was running a Ubuntu Linux 22.04 server image on UTM Version 3.2.4 (58) with no problems for many months.
Installed UTM Version 4.0.8 (67) the same Ubuntu Linux 22.04 server (same exact image) cannot get out to the network (network unreachable).

** UTM Version **
* UTM Version 4.0.8 (67)
* macOS 12.6
* Apple M1 Max 

** Debug Log **

Running:  -L /Applications/UTM.app/Contents/Resources/qemu -S -spice "unix=on,addr=/Users/gforghetti/Library/Group Containers/WDNLXAD4W8.com.utmapp.UTM/3FDF7032-92E6-4F56-BE53-D4F6B1D9AD1D.spice,disable-ticketing=on,image-compression=off,playback-compression=off,streaming-video=off,gl=off" -chardev spiceport,id=org.qemu.monitor.qmp,name=org.qemu.monitor.qmp.0 -mon chardev=org.qemu.monitor.qmp,mode=control -nodefaults -vga none -device virtio-net-pci,mac=92:DE:6D:1D:4E:31,netdev=net0 -netdev vmnet-bridged,id=net0,ifname=en5 -nographic -chardev spiceport,id=term0,name=com.utmapp.terminal.0 -serial chardev:term0 -cpu host -smp cpus=4,sockets=1,cores=4,threads=1 -machine virt-7.0 -accel hvf -drive if=pflash,format=raw,unit=0,file=/Applications/UTM.app/Contents/Resources/qemu/edk2-aarch64-code.fd,readonly=on -drive "if=pflash,unit=1,file=/Users/gforghetti/UTM_Images_Backup/Ubuntu 22.04.utm/Data/efi_vars.fd" -m 12288 -device intel-hda -device hda-duplex -device nec-usb-xhci,id=usb-bus -device usb-tablet,bus=usb-bus.0 -device usb-mouse,bus=usb-bus.0 -device usb-kbd,bus=usb-bus.0 -device qemu-xhci,id=usb-controller-0 -chardev spicevmc,name=usbredir,id=usbredirchardev0 -device usb-redir,chardev=usbredirchardev0,id=usbredirdev0,bus=usb-controller-0.0 -chardev spicevmc,name=usbredir,id=usbredirchardev1 -device usb-redir,chardev=usbredirchardev1,id=usbredirdev1,bus=usb-controller-0.0 -chardev spicevmc,name=usbredir,id=usbredirchardev2 -device usb-redir,chardev=usbredirchardev2,id=usbredirdev2,bus=usb-controller-0.0 -device virtio-blk-pci,drive=drive0,bootindex=0 -drive "if=none,media=disk,id=drive0,file=/Users/gforghetti/UTM_Images_Backup/Ubuntu 22.04.utm/Data/data.qcow2,discard=unmap,detect-zeroes=unmap" -device usb-storage,drive=drive1,removable=true,bootindex=1,bus=usb-bus.0 -drive if=none,media=cdrom,id=drive1 -name "Ubuntu 22.04" -uuid 3FDF7032-92E6-4F56-BE53-D4F6B1D9AD1D -rtc base=localtime -device virtio-rng-pci
qemu-aarch64-softmmu: warning: Spice: playback:0 (0x1430d6940): setsockopt failed, Operation not supported on socket
qemu-aarch64-softmmu: warning: Spice: record:0 (0x1430d69e0): setsockopt failed, Operation not supported on socket

**Upload VM**  
* It's very large and contains sensitive information

** Some Debug info **

** Ubuntu Linux 22.04 console running UTM Version 4.0.8 **

///////////////////////////////////////////////////////////////////////////////////////////////////////
// UTM Version 4.0.8 (67)
///////////////////////////////////////////////////////////////////////////////////////////////////////

```bash
root@ubuntu-machine:[~]# ping 8.8.8.8
ping: connect: Network is unreachable

root@ubuntu-machine:[~]# ip link show
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
2: enp0s1: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN mode DEFAULT group default qlen 1000
    link/ether 92:de:6d:1d:4e:31 brd ff:ff:ff:ff:ff:ff
3: virbr0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN mode DEFAULT group default qlen 1000
    link/ether 52:54:00:11:ac:67 brd ff:ff:ff:ff:ff:ff

root@ubuntu-machine:[~]# grep -r 'enp0s5' /etc/*
firewalld/zones/public.xml:  <interface name="enp0s5"/>
netplan/00-installer-config.yaml:    enp0s5:

root@ubuntu-machine:[~]# cat /etc/netplan/00-installer-config.yaml 
# This is the network config written by 'subiquity'
network:
  ethernets:
    enp0s5:
      dhcp4: true
  version: 2
root@ubuntu-machine:[~]# 
```

** Ubuntu Linux 22.04 console running UTM Version 3.2.4 **

///////////////////////////////////////////////////////////////////////////////////////////////////////
// UTM Version 3.2.4 (58)
///////////////////////////////////////////////////////////////////////////////////////////////////////

```bash
root@ubuntu-machine:[~]# ping 8.8.8.8
PING 8.8.8.8 (8.8.8.8) 56(84) bytes of data.
64 bytes from 8.8.8.8: icmp_seq=1 ttl=57 time=22.8 ms
64 bytes from 8.8.8.8: icmp_seq=2 ttl=57 time=20.2 ms
64 bytes from 8.8.8.8: icmp_seq=3 ttl=57 time=17.5 ms
^C
--- 8.8.8.8 ping statistics ---
3 packets transmitted, 3 received, 0% packet loss, time 2007ms

root@ubuntu-machine:[~]# ip link show
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
2: enp0s5: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP mode DEFAULT group default qlen 1000
    link/ether 92:de:6d:1d:4e:31 brd ff:ff:ff:ff:ff:ff
3: virbr0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN mode DEFAULT group default qlen 1000
    link/ether 52:54:00:11:ac:67 brd ff:ff:ff:ff:ff:ff

root@ubuntu-machine:[~]# grep -r 'enp0s5' /etc/*
/etc/firewalld/zones/public.xml:  <interface name="enp0s5"/>
/etc/netplan/00-installer-config.yaml:    enp0s5:

root@ubuntu-machine:[~]# cat /etc/netplan/00-installer-config.yaml 
# This is the network config written by 'subiquity'
network:
  ethernets:
    enp0s5:
      dhcp4: true
  version: 2
root@ubuntu-machine:[~]# 
```

**Comment by osy** [comment, marked as answer]
https://docs.getutm.app/updates/v4.0/#notes

**Reply by canoe-nc** [reply]
Ah. Okay.  I have to upgrade to macOS 13 Ventura in order to run UTM 4.x.
I am satisfied with this.  Closing this issue.

**Reply by osy** [reply]
That’s… not the takeaway.

**Reply by canoe-nc** [reply]
I understand.  I saw the requirement to have to modfiy the **/etc/netplan/00-installer-config.yaml** file with the correct interface name,  as well as UTM 4 changes the image so it is not compatible with UTM 3, so a backup of the image is a good idea.

**Comment by gadio12** [comment]
bvn
